diff --git a/node_modules/antd/es/collapse/Collapse.d.ts b/node_modules/antd/es/collapse/Collapse.d.ts
index 66f3294..e6a55b2 100644
--- a/node_modules/antd/es/collapse/Collapse.d.ts
+++ b/node_modules/antd/es/collapse/Collapse.d.ts
@@ -2,6 +2,8 @@ import * as React from 'react';
 import type { CollapseProps as RcCollapseProps } from 'rc-collapse';
 import type { SizeType } from '../config-provider/SizeContext';
 import type { CollapsibleType } from './CollapsePanel';
+import type { CSSMotionProps } from 'rc-motion';
+
 /** @deprecated Please use `start` | `end` instead */
 type ExpandIconPositionLegacy = 'left' | 'right';
 export type ExpandIconPosition = 'start' | 'end' | ExpandIconPositionLegacy | undefined;
@@ -26,6 +28,7 @@ export interface CollapseProps extends Pick<RcCollapseProps, 'items'> {
      * @deprecated use `items` instead
      */
     children?: React.ReactNode;
+    motion?: CSSMotionProps
 }
 interface PanelProps {
     isActive?: boolean;
diff --git a/node_modules/antd/es/collapse/Collapse.js b/node_modules/antd/es/collapse/Collapse.js
index 0591d39..b87902b 100644
--- a/node_modules/antd/es/collapse/Collapse.js
+++ b/node_modules/antd/es/collapse/Collapse.js
@@ -29,7 +29,8 @@ const Collapse = /*#__PURE__*/React.forwardRef((props, ref) => {
     size: customizeSize,
     expandIconPosition = 'start',
     children,
-    expandIcon
+    expandIcon,
+    motion,
   } = props;
   const mergedSize = useSize(ctx => {
     var _a;
@@ -71,7 +72,8 @@ const Collapse = /*#__PURE__*/React.forwardRef((props, ref) => {
   }, collapse === null || collapse === void 0 ? void 0 : collapse.className, className, rootClassName, hashId, cssVarCls);
   const openMotion = Object.assign(Object.assign({}, initCollapseMotion(rootPrefixCls)), {
     motionAppear: false,
-    leavedClassName: `${prefixCls}-content-hidden`
+    leavedClassName: `${prefixCls}-content-hidden`,
+    ...motion,
   });
   const items = React.useMemo(() => children ? toArray(children).map((child, index) => {
     var _a, _b;
diff --git a/node_modules/antd/es/notification/useNotification.js b/node_modules/antd/es/notification/useNotification.js
index f167e85..eb66e77 100644
--- a/node_modules/antd/es/notification/useNotification.js
+++ b/node_modules/antd/es/notification/useNotification.js
@@ -54,7 +54,9 @@ const Holder = /*#__PURE__*/React.forwardRef((props, ref) => {
     rtl,
     onAllRemoved,
     stack,
-    duration
+    duration,
+    pauseOnHover = true,
+    showProgress,
   } = props;
   const {
     getPrefixCls,
@@ -80,7 +82,8 @@ const Holder = /*#__PURE__*/React.forwardRef((props, ref) => {
     closable: true,
     closeIcon: getCloseIcon(prefixCls),
     duration: duration !== null && duration !== void 0 ? duration : DEFAULT_DURATION,
-    pauseOnHover: true,
+    pauseOnHover,
+    showProgress,
     getContainer: () => (staticGetContainer === null || staticGetContainer === void 0 ? void 0 : staticGetContainer()) || (getPopupContainer === null || getPopupContainer === void 0 ? void 0 : getPopupContainer()) || document.body,
     maxCount,
     onAllRemoved,
