diff --git a/node_modules/rc-collapse/es/Collapse.js b/node_modules/rc-collapse/es/Collapse.js
index 7877573..14c331c 100644
--- a/node_modules/rc-collapse/es/Collapse.js
+++ b/node_modules/rc-collapse/es/Collapse.js
@@ -29,7 +29,7 @@ var Collapse = /*#__PURE__*/React.forwardRef(function (props, ref) {
     className = props.className,
     children = props.children,
     collapsible = props.collapsible,
-    openMotion = props.openMotion,
+    openMotion = {},
     expandIcon = props.expandIcon,
     rawActiveKey = props.activeKey,
     defaultActiveKey = props.defaultActiveKey,
diff --git a/node_modules/rc-collapse/es/Panel.js b/node_modules/rc-collapse/es/Panel.js
index 3510471..876cc0c 100644
--- a/node_modules/rc-collapse/es/Panel.js
+++ b/node_modules/rc-collapse/es/Panel.js
@@ -61,7 +61,7 @@ var CollapsePanel = /*#__PURE__*/React.forwardRef(function (props, ref) {
     className: headerClassName,
     'aria-expanded': isActive,
     'aria-disabled': disabled,
-    onKeyDown: handleKeyDown,
+    // onKeyDown: handleKeyDown,
     style: styles.header
   };
   if (!collapsibleHeader && !collapsibleIcon) {
