diff --git a/node_modules/vite/dist/node/chunks/dep-Dyl6b77n.js b/node_modules/vite/dist/node/chunks/dep-Dyl6b77n.js
index 980fe44..470c30f 100644
--- a/node_modules/vite/dist/node/chunks/dep-Dyl6b77n.js
+++ b/node_modules/vite/dist/node/chunks/dep-Dyl6b77n.js
@@ -50100,6 +50100,24 @@ async function createDepsOptimizer(config, server) {
           timestamp: true
         }
       );
+      
+      const cacheFile = path.resolve(process.cwd(), process.env.VITE_OPTIMIZE_DEPS_FILE)
+
+      if (!fs.existsSync(cacheFile)) {
+        // init file
+        fs.writeFileSync(cacheFile, '')
+      }
+      // read file
+      const content = fs.readFileSync(cacheFile, 'utf-8')
+      const lines = content.split('\n')
+      const deps = lines.filter((line) => line.trim() !== '').map((line) => line.trim())
+
+      const allDeps = new Set([...deps, ...newDepsToLog])
+      fs.writeFileSync(cacheFile, [...allDeps].join('\n'))
+      logger.info(colors$1.green(`[image-manager]: add deps to ${process.env.VITE_OPTIMIZE_DEPS_FILE}`), {
+         timestamp: true
+      })
+
       newDepsToLog = [];
     }
   };
